app/views/WhatYouOwe.scala.html

```diff
@@ -287,7 +289,11 @@
 @tableRow(rowId2: String, documentDetailWithDueDate: DocumentDetailWithDueDate, isOverduePayment: Boolean, index:Int, id:String) = {
     <tr class="govuk-table__row" id="@rowId2">
         <td class="govuk-table__cell govuk-!-width-one-quarter" >
-            @documentDetailWithDueDate.documentDetail.documentDueDate.getOrElse(throw MissingFieldException("document due date of Payment")).toLongDateShort
+            @if(documentDetailWithDueDate.documentDetail.isOtherInterest && documentDetailWithDueDate.documentDetail.isPaid){
```

Comment:
we would need to update https://jira.tools.tax.service.gov.uk/browse/MISUV-8352 with more details after merge

Reply:
Ok, will do